client/src/plugins/libparsec/definitions.ts

```diff
@@ -691,6 +699,12 @@ export interface LibParsecPlugin {
         recipient: UserID,
         role: RealmRole | null
     ): Promise<Result<null, ClientWorkspaceShareError>>
+    clientInfo(
+        client: number
+    ): Promise<Result<[OrganizationID, DeviceID, UserID, UserProfile, HumanHandle | null], ClientWorkspaceInfoError>>
```

Comment:
Could you create an interface? I don't want to check the function prototype to know in which order the elements are given, I should just be able to do `result.value.organizationId`, not `result.value[0]`.

Reply:
Related https://github.com/Scille/parsec-cloud/pull/5198#discussion_r1324282525

Reply:
Done